aci/types/linked_accounts.py

```
@@ -37,6 +37,23 @@ class LinkedAccountUpdate(BaseModel):
     enabled: bool | None = None
 
 
+class NoAuthSchemeCredentials(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)